cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -72,6 +71,7 @@ func sampleUnaryMethod(theX int64, theB string, theFoo string, bobFile string) e
 				Y: 3,
 			},
 		},
+		DataBob: bobFileBytes,
```

Comment:
We used to have `defaults` and `attributes` in the request configuration items in these two directives do not have to be in the same order. But now default value and sample argument name are configured in the sample directive (`request.value` and `request.input_parameter`), thus causing poor `DataBob` to be moved to the end.